### ISSUE TITLE
Add an NU5 block test vector that contains many shielded Orchard transactions

### ISSUE THREAD

**Issue by dconnolly** [ZcashFoundation/zebra#4774]
## Motivation

When blocks started showing up on mainnet after NU5 with many shielded Orchard transactions, we observed aberrant zebrad verification behavior that helped us identify some [bugs](https://github.com/ZcashFoundation/zebra/pull/4752) already. We should include these blocks (at least one) as test vectors.

## Related Work

#4775

**Comment by str4d** [comment]
I wrote a script for generating (most of) the necessary data to benchmark verification of specific heavy blocks (the heaviest I could find at their respective times) using a mainnet `zcashd`, which we use to avoid checking in a large amount of data that is only used for performance benchmarks. The script collects the noted blocks and their transparent inputs (in `zcashd`'s coins LevelDB format), and writes out a tarball. We then pair that with some hard-coded Sapling and Orchard trees (that I extracted by hand, as the RPCs didn't quite have enough expressiveness to expose them) that are needed to fully-validate the blocks.

- https://github.com/zcash/zcash/blob/master/qa/zcash/create_benchmark_archive.py
- https://github.com/zcash/zcash/blob/882935e35ee338876c071f9cd93aa3694154de73/src/zcbenchmarks.cpp#L436-L702

(For reference, the usage of these in our benchmarks is `./qa/zcash/performance-measurements.sh time connectblockslow` or `connectblocksapling / connectblockorchard`)
